### DISCUSSION TITLE
Cannot find module calling remote component using typescript

### DISCUSSION THREAD

**Discussion by imadueno** [module-federation/module-federation-examples#2171] [Q&A]
Hello everyone.

I came to you asking for help.
I'm creating an module federated app using:
- nextjs 12
- react 18
- module federation/nextjs-mf 4.1.0
- typescript v4.7.2

I'm trying to achieve this.
![image](https://user-images.githubusercontent.com/29583359/182917290-23c7670e-8164-41f6-9368-11da980f2977.png)

[This example](https://github.com/module-federation/module-federation-examples/tree/master/react-nextjs/nextjs-host-remote/host) uses two nextjs apps without typescript, wich work perfectly.

With the same host app ( wich is the main app ), exposed a component that I receive in a regular nextjs app, and thats great.

## The problem 

Using the same host configuration as the documentation example, I'm getting this error on `pages/index.tsx`

![image](https://user-images.githubusercontent.com/29583359/182916502-0fb3b042-800f-423e-8398-b68dcdbfdfe3.png)

I have days searching and trying to figure out why it's happening, tryed many `.eslintrc`-`.eslintrc.json` configurations and packages and nothing fix the problem.

Has anyone been through this before?

**Comment by ScriptedAlchemy** [comment]
https://github.com/module-federation/core/issues/3153
